### PR TITLE
Switch to aktualizr 2020.9

### PR DESCRIPTION
**Pull request by mike-sul**

### CODE REVIEW

**Comment by doanac** [comment]
i'm going to pull this into my factory and do some testing either today or tomorrow.

File: unit-test [hunk 1]
```diff
@@ -14,7 +14,7 @@ build=$(mktemp -d -p $here)
 cd $build
 
 # do clang-tidy during compilation
-../cmake-init.sh -DBUILD_WITH_CLANG_TIDY=ON
+../cmake-init.sh
```
Comment (doanac) [inline unit-test:17]: should we really remove the ability to run clang tidy, or just disable it by default for CI until we can fix?

Comment (mike-sul) [reply unit-test:17]: I just removed the clang tidy verification that happens during the source code building, I added it initially and it's kind of duplication of what the aktualizr clang tidy functionality does. So there is still clang tidy the one that aktualizr provides.

**Comment by doanac** [comment]
looks sane. i'll give this a test tomorrow

**Comment by mike-sul** [comment]
> looks sane. i'll give this a test tomorrow

Thanks, I'll do the same today too.

**Pull request merged**